desktop/ELECTRON_ENVIRONMENT.js

```diff
@@ -1,19 +1,10 @@
-// This variable is injected into package.json by electron-builder via the extraMetadata field (specified in electronBuilder.ghactions.config.js)
-// It will be `PROD` on production, `STG` on staging, and `undefined` on dev (because dev doesn't use electron-builder)
-const {electronEnvironment} = require('../package.json');
 const ENVIRONMENT = require('../src/CONST/ENVIRONMENT');
 
 /**
- * @returns {String} – One of ['PROD', 'STG', 'DEV']
+ * @returns {'development'|'staging'|'production'}
  */
 function getEnvironment() {
-    // If we are on dev, then the ELECTRON_ENVIRONMENT environment variable will be present (set in package.json desktop script `npm run desktop`)
-    if (process.env.ELECTRON_ENV === ENVIRONMENT.DEV) {
-        return ENVIRONMENT.DEV;
-    }
-
-    // Otherwise, use the environment injected into package.json by electron-builder
-    return electronEnvironment;
+    return process.env.NODE_ENV || ENVIRONMENT.DEV;
```

Comment:
This was substituted for `NODE_ENV` because: 
1. `const {electronEnvironment} = require('../package.json')` - now that `main.js` is bundled by webpack this line gets evaluated before `electronEnvironment` is added to `package.json` 
2. we can use `process.env.ELECTRON_ENV`, but we need to tell webpack about it (DefinePlugin)
3. We're only interested in `isDev` vs `isProd` here and `NODE_ENV` serves OK for this

Reply:
Sorry, where is `NODE_ENV` set here? This is used by the Electron main process, right?